spec/generator/project_spec.rb

```diff
@@ -21,6 +21,28 @@ module Generator
         expect(@generator.project).not_to be_nil
       end
 
+      it 'should set class prefix' do
+        prefix = "XC"
+
+        allow(@project).to receive(:class_prefix).and_return(prefix)
+        @generator.visit_project(@project)
+
+        expect(@generator.project.attributes['CLASSPREFIX']).to eq(prefix)
+      end
+
+      it 'should set organization name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.